guides/03_API_Overview.md

```diff
@@ -38,6 +38,10 @@ Using the above objects, create a [DefaultMeetingSession](https://aws.github.io/
 const meetingSession = new DefaultMeetingSession(configuration, logger, deviceController);
 ```
 
+### 1e. Create a meeting from your server application
+
+In your meeting service, you can create a Chime object without the Amazon Chime SDK for JavaScript. See [Getting responses from your server application](https://github.com/aws/amazon-chime-sdk-js#getting-responses-from-your-server-application) for more information.
```

Comment:
We cannot create the Chime object using Amazon Chime SDK for JavaScript right? We use AWS SDK to create the Chime object.

Should we reframe this to : Use AWS SDK to create the Chime object. Use the Chime object to get meeting and attendee objects from [CreateMeeting](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateMeeting.html) and [CreateAttendee](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateAttendee.html) Chime APIs. See [Getting responses from your server application](https://github.com/aws/amazon-chime-sdk-js#getting-responses-from-your-server-application) for more information.